src/plugins/lvm.h

```diff
@@ -121,6 +144,19 @@ typedef struct BDLVMLVdata {
 void bd_lvm_lvdata_free (BDLVMLVdata *data);
 BDLVMLVdata* bd_lvm_lvdata_copy (BDLVMLVdata *data);
 
+typedef struct BDLVMVDOPooldata {
+    BDLVMVDOOperatingMode operating_mode;
+    BDLVMVDOCompressionState compression_state;
+    BDLVMVDOIndexState index_state;
+    guint64 used_size;
+    guint64 saving_percent;
+    gboolean deduplication;
+    gboolean compression;
+} BDLVMVDOPooldata;
```

Comment:
Also, additional VDO statistics are not (and won't be) covered by lvm and users are told to use `vdostats`. We have this in the vdo plugin (`bd_vdo_get_stats` and `bd_vdo_get_stats_full`), should we copy the code here or tell users to use the vdo plugin? Also the question is if we want to keep the vdo plugin in 3.0 or just remove it and tell users to use vdo only via LVM.

Reply:
The VDO plugin will get deprecated one day I suppose, you may want to separate the code into a new source file and include at both places.

Reply:
Let's do that for 3.0.